packages/frontend/cypress.json

```diff
@@ -0,0 +1,28 @@
+{
+    "baseUrl": "http://localhost:3000",
+    "userAgent": "synpress",
+    "retries": {
+        "runMode": 0,
+        "openMode": 0
+    },
+    "integrationFolder": "cypress/integration/specs",
+    "screenshotsFolder": "cypress/screenshots",
+    "videosFolder": "cypress/videos",
+    "supportFile": "cypress/support/index.js",
+    "pluginsFile": "cypress/plugins/index.js",
+    "chromeWebSecurity": true,
+    "viewportWidth": 1920,
+    "viewportHeight": 1080,
+    "scrollBehavior": false,
+    "component": {
+        "componentFolder": ".",
+        "testFiles": "**/*.spec.{js,jsx,ts,tsx}"
+    },
+    "env": {
+        "coverage": true
+    },
+    "defaultCommandTimeout": 30000,
+    "pageLoadTimeout": 30000,
+    "requestTimeout": 30000,
+    "projectId": "kbncm6"
```

Comment:
What projectId means?

Reply:
This helps the cypress dashboard uniquely identify the project, we can integrate the dashboard when the CI is ready, will remove it for now